src/utils/Util.js

```diff
@@ -8,12 +8,23 @@ const remote = electron.remote;
 const dialog = remote.dialog;
 const app = remote.app;
 
+// Docker Machine exits with 3 to differentiate pre-create check failures (e.g.
+// virtualization isn't enabled) from normal errors during create (exit code
+// 1).
+const precreateCheckExitCode = 3;
+
 module.exports = {
+  errPrecreateCheckFailure:  'Pre-create check failed: ',
   execFile: function (args, options) {
+    var that = this;
```

Comment:
Should be unnecessary given that we use fat arrow functions. I.e. the  `=> {`  syntax.